app/models/ability.rb

```diff
@@ -56,6 +56,9 @@ def school_manager_abilities(user:)
       can [:manage_school_users], School do |school|
         school.id == user.school_id
       end
+      can [:manage_school_students], School do |school|
+        school.id == user.school_id
```

Comment:
might be easier by comparing user.school_id

Reply:
not really an option ; on index, we check if the user can access a school users, not the user specificaly